apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/DatabaseSchemaManager.java

```diff
@@ -24,6 +24,7 @@
  * Liquibase changelog is in <code>src/main/resources/db-schema.xml</code>.
  */
 public final class DatabaseSchemaManager {
+  /** Outputs information to the log file. */
```

Comment:
still have a logger comment here and in two other places

Reply:
Fixed the remaining spots!